CHANGELOG.md

```diff
@@ -1,3 +1,54 @@
+Version 4.3.0
+===========
+
+Breaking Changes
+----------------
+* Raise Minimum Python Version to 3.8 (#1597)
+
+Bug Fixes
+---------
+* Fix Vector channel detection (#1634)
+* ixxat: Fix exception in 'state' property on bus coupling errors (#1647)
+* NeoVi: Fixed serial number range (#1650)
+* Fix PCAN timestamp (#1651)
+* Fix BLFReader error for incomplete or truncated stream (#1662)
+* We do not need to account for drift when we `USE_WINDOWS_EVENTS` (#1666)
+
+Features
+--------
+
+### IO
+* Distinguish Text/Binary-IO for Reader/Writer classes. (#1585)
+* Can Player compatibility with interfaces that use additional configuration (#1610)
+
+### Interface Improvements
+* Change Bus constructor implementation and typing (#1557)
+* Add `protocol` property to BusABC to determine active CAN Protocol (#1532)
+* Add auto-modifying cyclic tasks (#703)
```

Comment:
```suggestion
* Add `modifier_callback` parameter to `BusABC.send_periodic` for auto-modifying cyclic tasks (#703)
```

Add an "API" sub section to "Features", there are a few points here, which i'd move to API